src/dynamo.js

```diff
@@ -16,7 +16,8 @@ export class Dynamo {
         lastHeartbeat: new Date().getTime()
       })
     }
-    await client.putItem(params)
+    const result = await client.putItem(params)
+    console.log(result)
```

Comment:
Is `result` self-describing? Should we instead do something like `console.log("DynamoDB result", result")`?

Reply:
Sorry. We tried something with Slavi and I forgot to delete it. Actually, we don't need console.log().